### ISSUE TITLE
New song notification

### ISSUE THREAD

**Issue by oisterjosh** [RKGman/fire-beats#7]
Could a toggle be added to stop the plugin from adding a notifaction to the "game" tab? It is a bit of spam in my game chat, and sometimes tricks me mid-olm etc, thinking the "the teleport does nothing" notification popped.

**Comment by RKGman** [comment]
I do need to leave it in for credit to the artists; however, I think there is probably a way to prevent "spam" / repeated messages.  I will look into it.

**Comment by oisterjosh** [comment]
fair enough. I would rather a tab on the side for "now playing" and possibly a toggleable overlay. I've had to stop using the plugin altogether as every new song load causes large delays and is too problematic for pvm

**Comment by HalfOfAKebab** [comment]
As a workaround until this is properly fixed, the Chat Filter plugin can be used to filter the messages.

Enable the Chat Filter plugin, tick the "Filter Game Chat" option, and paste this into the Filtered Regex box: `^fire beats notice:`
